rules/Proj0043.md

```diff
@@ -0,0 +1,76 @@
+---
+parent: General
+ancestor: MSBuild
+---
+
+# Proj0043: Enable &lt;RestoreLockedMode&gt; when &lt;ContinuousIntegrationBuild&gt; is enabled
+In order to ensure `dotnet restore` is not allowed to implicitly upgrade package versions that are
+specified in the [lock file](https://devblogs.microsoft.com/dotnet/enable-repeatable-package-restores-using-a-lock-file/). It is necessary to enable "locked mode".
+The [recommended way](https://devblogs.microsoft.com/dotnet/enable-repeatable-package-restores-using-a-lock-file/)
+of enabling locked mode is to conditionally set the `RestoreLockedMode` property to `true`,
+if the `ContinuousIntegrationBuild` is set to `true`. This is under the assumption that
+[there is a dynamic way in place that ensures `ContinuousIntegrationBuild` is set to `true` in CI pipelines](./Proj0044.md).
```

Comment:
dynamic way?

Reply:
I couldn't think of a better way to describe it 😅